.travis.yml

```diff
@@ -1,14 +1,15 @@
 language: node_js
 node_js:
   - "6"
-  - "4"
 
-# before_install:
-  # - apt-get -y install python-pip
-  # - pip install --upgrade pip
+sudo: required
+
+before_install:
+  - cd $TRAVIS_BUILD_DIR
 
 install:
-  - ./tools/travis/setupscan.sh
+  - ./tools/travis/unitsetup.sh
+  - cd $TRAVIS_BUILD_DIR
 
 script:
-  - ./tools/travis/scancode.sh
\ No newline at end of file
+  - ./tools/travis/unit.sh
```

Comment:
Scancode run in unit.sh
```
# run scancode
cd $UTILDIR
scancode/scanCode.py $ROOTDIR
```
But good to point out I should remove extraneous files. I forgot those when I copied stuff from one branch over to this one.